### DISCUSSION TITLE
[New Feature] Underground tunnels for xenomorphs

### DISCUSSION THREAD

**Discussion by SoldierTODD** [Vinifera-Developers/Vinifera#1492] [Feature Requests]
### Description:
Please add the function of underground tunnels / burrows under the corresponding animation of entry and exit. 

### Possible Implementation:
Example:
![14](https://user-images.githubusercontent.com/32884056/120108341-6e3cb380-c16d-11eb-94c9-aed1056f83c1.gif)
![15](https://user-images.githubusercontent.com/32884056/120108342-6ed54a00-c16d-11eb-84a8-864dba78cfc4.gif)

**Comment by Rampastring** [comment]
Would this be any different from the subterranean movement logic used by the Devil's Tongue?

**Comment by Metadorius** [comment]
> Would this be any different from the subterranean movement logic used by the Devil's Tongue?

He talks about SHP animation support for burrowing/unburrowing.

**Comment by SoldierTODD** [comment]
`Would this be any different from the subterranean movement logic used by the Devil's Tongue?`

I think no.
It would be great if there were multiple entrances and exits from an underground tunnel / burrow (like a subway network). So that the player can choose the exit from the underground tunnel.
